recipes/tapestry/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+mkdir -p $PREFIX/bin
+cp $SRC_DIR/weave $PREFIX/bin
+cp $SRC_DIR/clean $PREFIX/bin
```

Comment:
Is there a reason that installing the package doesn't install these?

Reply:
Thanks for the rapid replies! Just my ignorance, I have now updated the setup.py to include the scripts and removed these lines from build.sh.